Meta/.shell_include.sh

```diff
@@ -13,3 +13,26 @@ die() {
     echo "die: $*"
     exit 1
 }
+
+find_executable() {
+  paths=("/usr/sbin/" "/sbin/")
+  executable="$1"
+
+  # Prefer tools from PATH over fallback paths
+  if command -v "$executable"; then
+    return 0
+  fi
+
+  for path in "${paths[@]}"; do
+    if command -v "$path$executable"; then
```

Comment:
Ultimate nitpick, but I'd prefer omitting the trailing slashes from the paths array and using `$path/$executable` instead.

Also, while we're at it, adding curly braces around variable names (i.e. `${path}/${executable}`) would be great for readability. The same probably applies for usages of `${executable}` alone as well.